docs/DeveloperGuide.md

```diff
@@ -373,6 +383,7 @@ A flow of a client class using the parsing services provided by `DateUtil` for a
 
 ![Parser Flow](images/DateTimeParsingSequenceDiagram.png)
 
+
```

Comment:
Please remove this addition, I don't think you should be editing this section. Thanks.

Reply:
thx removed